docs/pages/en-us/Client-API.md

```diff
@@ -140,13 +144,13 @@ The `AuditConnection` object can be constructed using the static `createConnecti
 
 An `AuditChannel` object is constructed from the [`AuditConnection`](#AuditConnection) object.
 
-This object represents a logical channel to the persistent storage (that is, Kafka in this implementation). ***It is NOT a thread-safe object and must not be shared across threads without synchronization.*** However, you will have no issue constructing multiple `AuditChannel` objects simultaneously on different threads. The objects are lightweight and caching them is not that important.
+This object represents a logical channel to the persistent storage (that is, Elasticsearch in this implementation). ***It is NOT a thread-safe object and must not be shared across threads without synchronization.*** However, you will have no issue constructing multiple `AuditChannel` objects simultaneously on different threads. The objects are lightweight and caching them is not that important.
```

Comment:
Just because I know we changed the Getting Started page.  
Replace, "the persistent storage"
With, "the datastore"